model/textparse/openmetricsparse_test.go

```diff
@@ -992,3 +993,46 @@ go_gc_duration_seconds_created`)
 	require.Equal(t, "go_gc_duration_seconds", string(copyParser.l.b[copyParser.offsets[0]:copyParser.offsets[1]]))
 	require.False(t, copyParser.skipCTSeries)
 }
+
+func BenchmarkOMParse(b *testing.B) {
+	for parserName, parser := range map[string]func([]byte, *labels.SymbolTable) Parser{
+		"openmetrics": func(b []byte, st *labels.SymbolTable) Parser {
+			return NewOpenMetricsParser(b, st)
+		},
+		"openmetrics-skip-ct": func(b []byte, st *labels.SymbolTable) Parser {
```

Comment:
Perhaps it would make sense here to always do openmetrics (no casing)but haveb.Run(fmt.Sprintf("skip-ct=%v", skipCT), ....)` case, WDYT?

Reply:
yes I agree with that bit
virtually no difference here for both parsers

Reply:
Happy to do in separare PR